<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task1Class</class>
 <widget class="QMainWindow" name="Task1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>371</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>543222</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Kosmin Pavel</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>564321</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>Vladimcev Vadim</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>234567</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Petrov Alexandr</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>345611</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>Ivanov Petr</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>868679</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>Sidorov Kirill</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>767565</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>Homenkov Oleg </string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>131242</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>Mamchenko Kirill</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="createTreeBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Tree</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableForTree">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>141</width>
      <height>371</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Tree Keys</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Traverse Inorder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Traverse Preorder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Traverse Postorder</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="changeTraverseBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Traverse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyLineEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="valueLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueLineEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyLineEdit_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getValueBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyLineEdit_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyLineEdit_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getNofSymbols">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Number of Symbols</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Task1.qrc"/>
 </resources>
 <connections/>
</ui>
